plugins/parsers/avro/testdata/supplied_timestamp_fields_unspecified/telegraf.conf

```diff
@@ -4,8 +4,6 @@
   avro_measurement = "measurement"
   avro_tags = [ "tag" ]
   avro_fields = [ "field" ]
-  avro_timestamp = "timestamp"
-  avro_timestamp_format = "unix_us"
```

Comment:
Why is this test modified? I think it tests for the case where you specify a timestamp field but that field is not contained in the message, isn't it?

Reply:
That test was originally to test that it would reject a specified timestamp format if the timestamp field wasn't defined.  But then, as a result of these changes, we just default the format to "unix" in general and use the time of receipt as the timestamp...so probably the test needs renaming or removing.

Reply:
Also, yeah, I will certainly rebase once #13855 is merged.  After this one there's one about handling union types, but we're going to need to test that one at Rubin in operation for a while before I feel like it's doing the right thing.

Reply:
Now rebased.